components/swap-popup.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import stylesheet from './swap-popup.scss'
+import buttonStyleSheet from './button.scss'
+import { Media } from './media'
+import { Button } from './button'
+
+const cookieName = "mybit_swap_popup_never";
+
+export class SwapPopup extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.state = {popup: false}
+    this.handleClickDontShow = this.handleClickDontShow.bind(this);
+  }
+
+  componentDidMount(){
+    if(document.cookie.indexOf(cookieName) === -1){
+      this.setState({popup: true})
+    }
+  }
+
+  handleClickDontShow(){
+    document.cookie = `${cookieName}=true; expires=Fri, 31 Dec 9999 23:59:59 GMT`;
+    this.setState({popup: false})
+  }
+
+  render(){
+    if(this.state.popup){
+      const content = (
+        <div className="SwapPopup__content">
+          <style dangerouslySetInnerHTML={{ __html: buttonStyleSheet }} />
+          <p>The MyBit Token Swap is now live! The Token Swap will be exchanging current MyBit Tokens with a new, updated ERC20 token, also with the symbol MYB. For more information on the Token Swap please read our blog post by clicking ‘See Why’. To view a comprehensive guide and get started please click ‘Swap Now’.</p>
+          <div className="SwapPopup__buttons">
```

Comment:
Probably need to do something like:

```scss
  &__buttons {
    display: flex;
    justify-content: center;
    margin: 20px 0;
  }
  &__button {
    margin: 0 5px;
  }
```

and add `SwapPopup__button` to the `a` element